inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -62,6 +62,9 @@ void LayerTestsCommon::Run() {
         s.updateOPsStats(function, PassRate::Statuses::FAILED);
         GTEST_FATAL_FAILURE_("Unknown failure occurred.");
     }
+    if (s.getSaveReportWithUniqueName()) {
```

Comment:
In such case you don't need to save the report in environment teardwown, please add an additional check there

Reply:
fixed